.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MESSAGES CONTROL]
+
+disable=broad-except,
+        too-few-public-methods,
+        parse-error,
+        invalid-name,
+        attribute-defined-outside-init,
+        pointless-statement,
+        useless-object-inheritance,
+        no-else-return,
+        no-name-in-module,
+        len-as-condition,
+        missing-docstring,
+        bad-whitespace,
+        duplicate-code,
+        unused-import,
+        unused-variable,
+        useless-else-on-loop,
+        bad-continuation,
+        import-error,
+        too-many-arguments,
+        logging-not-lazy,
+        undefined-variable,
```

Comment:
This is only needed, because `requirements.txt` is linted.
```
requirements.txt:1:0: E0602: Undefined variable 'pillow' (undefined-variable)
requirements.txt:2:0: E0602: Undefined variable 'polib' (undefined-variable)
requirements.txt:3:0: E0602: Undefined variable 'requests' (undefined-variable)
requirements.txt:4:0: E0602: Undefined variable 'radon' (undefined-variable)
requirements.txt:5:0: E0602: Undefined variable 'urllib3' (undefined-variable)
requirements.txt:6:0: E0602: Undefined variable 'xmlschema' (undefined-variable)
```